projects/stb/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+$CXX $CXXFLAGS -std=c++11 -I. \
+    $SRC/stb/tests/stb_png_read_fuzzer.cpp \
+    -o $OUT/stb_png_read_fuzzer $LIB_FUZZING_ENGINE
+
+find $SRC/stb/tests/pngsuite -name "*.png" | \
+     xargs zip $OUT/stb_png_read_fuzzer_seed_corpus.zip
+
+cp $SRC/stb/tests/stb_png.dict \
```

Comment:
This needs to stb_png_read_fuzzer.dict when you copy to $OUT.

Reply:
Looks like it's copied to the right path in the next commit (https://travis-ci.org/google/oss-fuzz/jobs/646927286#L504), also the .options file is not copied anymore, it only specified the dictionary path.